### PR TITLE
User signs out from account

### PR DESCRIPTION
**Pull request by koggs**
### What we learned
- solve merge conflicts

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -25,3 +21,12 @@
     create(:user)
   end
 end
+
+Given("I open the browser") do
```
Comment (tochman) [inline features/step_definitions/basic_steps.rb]: This name is a bit misleading. I would go with 
```ruby
Then("show me the page") do
```

File: features/user_logs_in_to_account.feature [hunk 2]
```diff
@@ -9,31 +9,31 @@ Background:
     | Maran   | maran@test.com  | 12345678    |
 
 Scenario: User can log in with valid credentials
-  Given I visit the site
+  Given I visit site
```
Comment (tochman) [inline features/user_logs_in_to_account.feature]: Not sure I'm onboard with this change.

File: features/user_signs_out_from_account.feature [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  In order to keep my account safe
+  I would like to be able to sign out
+
+Background:
+  Given We have the following user:
+    | email           | password    |
+    | maran@test.com  | 12345678    |
+
+Scenario: User signs out
+  Given I visit login page
```
Comment (tochman) [inline features/user_signs_out_from_account.feature]: I would add the word "the" ... Sounds better when read out loud IMO.

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (koggs) [reply features/step_definitions/basic_steps.rb]: That's already fixed

**Pull request merged**